### PR TITLE
Db command

### LINKED ISSUE #11: fix assertUserIsRegistered() in UserService
noticed that after a user is in the DB, other commands may result in the bot giving the message "It seems you're not registered. Type !start to register right away!". 

Commented out this line for the time-being. UserService (line 16)
 ```await this.assertUserIsRegistered();```

### PR DESCRIPTION
**Pull request by tayoogunbiyi**
This PR enables all the existing commands for the bot. Fixes #5 and fixes #11 as well.

### CODE REVIEW

File: commands/linkedin.js [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
-const run = (message, args) => {
-    message.author.send('Yay! We have set your linkedin profile to ' + args);
-}
+const run = async (message, args) => {
+    try {
+        await svc.updateByDiscordId(message.author.id, {
+            linkedin: args,
+        });
+        message.author.send(
+            `Yay! We have set your linkedin profile to \`${args}\``
```
[the remark this answers is not in the document]
Comment (tayoogunbiyi) [reply commands/linkedin.js:19]: I think it looks better when it displays as a 'link'. It's not exactly a link but just a nice highlighted background

[the remark this answers is not in the document]
Comment (smartclash) [reply commands/linkedin.js:19]: What do you mean by "It's not exactly a link"?

**Comment by tayoogunbiyi** [comment]
@smartclash changes have been applied  :)

File: commands/github.js [hunk 2]
```diff
@@ -15,9 +15,7 @@ const run = async (message, args) => {
         await svc.updateByDiscordId(message.author.id, {
             github: args,
         });
-        message.author.send(
-            `Yay! We have set your github profile to \`${args}\``
-        );
+        message.author.send('Yay! We have set your github profile to' + args);
```
Comment (smartclash) [inline commands/github.js:18]: A space is required.

```suggestion
        message.author.send('Yay! We have set your github profile to ' + args);
```

File: commands/funfact.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const run = async (message, args) => {
         await svc.updateByDiscordId(message.author.id, {
             funfact: args,
         });
-        message.author.send(`Yay! We have set your funfact as to \`${args}\``);
+        message.author.send('Yay! We have set your funfact as to' + args);
```
Comment (smartclash) [inline commands/funfact.js:18]: A space is required

```suggestion
        message.author.send('Yay! We have set your funfact as to'  + args);
```

File: commands/linkedin.js [hunk 4]
```diff
@@ -15,9 +15,7 @@ const run = async (message, args) => {
         await svc.updateByDiscordId(message.author.id, {
             linkedin: args,
         });
-        message.author.send(
-            `Yay! We have set your linkedin profile to \`${args}\``
-        );
+        message.author.send('Yay! We have set your linkedin profile to' + args);
```
Comment (smartclash) [inline commands/linkedin.js:18]: A space is required

```suggestion
        message.author.send('Yay! We have set your linkedin profile to ' + args);
```

**Pull request merged**